lib/services/platform-service.ts

```diff
@@ -368,15 +368,56 @@ export class PlatformService implements IPlatformService {
 				this.$errors.failWithoutHelp("Unable to find built application. Try 'tns build %s'.", platform);
 			}
 
-			this.$fs.ensureDirectoryExists(path.dirname(targetPath)).wait();
+			function ext(file: string): string {
+				return path.extname(file).toLowerCase();
+			}
+			function targetExt(file: string): string {
+				let e = ext(file);
+				return (e === ".ipa" || e === ".app") ? e : ".";
```

Comment:
this `.` confuses me, I've started thinking that you expect to pass `--copy-to .`, while in fact the dot means everything that's not `.ipa` or `.app`
Can we change it to something else (not a merge stopper, I'm just wondering if we can find a better name)